cluster/src/assembly/resources/conf/iotdb-cluster.properties

```diff
@@ -57,117 +57,117 @@ seed_nodes=127.0.0.1:9003,127.0.0.1:9005,127.0.0.1:9007
 default_replica_num=3
 
 # cluster name to identify different clusters
 # all node's cluster_name in one cluster are the same
-cluster_name=default
+# cluster_name=default
 
 # connection time out (ms) among raft nodes
-connection_timeout_ms=20000
+# connection_timeout_ms=20000
 
 # write operation timeout threshold (ms), this is only for internal communications,
 # not for the whole operation.
-write_operation_timeout_ms=30000
+# write_operation_timeout_ms=30000
 
 # read operation timeout threshold (ms), this is only for internal communications,
 # not for the whole operation.
-read_operation_timeout_ms=30000
+# read_operation_timeout_ms=30000
 
 # catch up timeout threshold (ms), this is used for a follower behind the leader too much,
 # so the leader will send logs(snapshot) to the follower,
 # NOTICE, it may cost minutes of time to send a snapshot,
 # so this parameter should be larger than the snapshot cost time.
-catch_up_timeout_ms=300000
+# catch_up_timeout_ms=300000
 
 # whether to use batch append entries in log catch up
-use_batch_in_catch_up=true
+# use_batch_in_catch_up=true
 
 # the minimum number of committed logs in memory, after each log deletion, at most such number of logs
 # will remain in memory. Increasing the number will reduce the chance to use snapshot in catch-ups,
 # but will also increase the memory footprint
-min_num_of_logs_in_mem=1000
+# min_num_of_logs_in_mem=1000
 
 # maximum number of committed logs in memory, when reached, a log deletion will be triggered.
 # Increasing the number will reduce the chance to use snapshot in catch-ups, but will also increase
 # memory footprint
-max_num_of_logs_in_mem=2000
+# max_num_of_logs_in_mem=2000
 
 # maximum memory size of committed logs in memory, when reached, a log deletion will be triggered.
 # Increasing the number will reduce the chance to use snapshot in catch-ups, but will also increase
 # memory footprint, default is 512MB
-max_memory_size_for_raft_log=536870912
+# max_memory_size_for_raft_log=536870912
 
 # deletion check period of the submitted log
-log_deletion_check_interval_second=-1
+# log_deletion_check_interval_second=-1
 
 # Whether creating schema automatically is enabled, this will replace the one in iotdb-engine.properties
-enable_auto_create_schema=true
+# enable_auto_create_schema=true
 
 # consistency level, now three consistency levels are supported: strong, mid, and weak.
 # Strong consistency means the server will first try to synchronize with the leader to get the
 # newest data, if failed(timeout), directly report an error to the user;
 # While mid consistency means the server will first try to synchronize with the leader,
 # but if failed(timeout), it will give up and just use current data it has cached before;
 # Weak consistency does not synchronize with the leader and simply use the local data
-consistency_level=mid
+# consistency_level=mid
 
 # Whether to use asynchronous server
-is_use_async_server=false
+# is_use_async_server=false
```

Comment:
Hi, the `useAsyncServer` is true in `ClusterConfig.java`, so if we commented this parameter out, the cluster will start as aysnc-mode, we'd better be able to make them consistent again.

Reply:
got it.

Reply:
The ci is broken, we should fix this. This endless ci is mainly because we do not handle the logic right in cluster tests on sync-mode. I partially fixed this problem in this [commit](https://github.com/apache/iotdb/commit/caa8bd6a75f6530734321e209d2339cdf52d3a4a), and further fixes are needed.

Reply:
fixed.